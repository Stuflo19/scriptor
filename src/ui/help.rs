use ratatui::{
    style::{Modifier, Style, Stylize},
    text::{Line, Text},
    widgets::Paragraph,
};

pub fn generate_help_text() -> Paragraph<'static> {
    let (msg, style) = (
        vec!["Press ".into(), "esc".bold(), " to exit, ".into()],
        Style::default().add_modifier(Modifier::RAPID_BLINK),
    );
    let text = Text::from(Line::from(msg)).patch_style(style);
    
    Paragraph::new(text)
}
