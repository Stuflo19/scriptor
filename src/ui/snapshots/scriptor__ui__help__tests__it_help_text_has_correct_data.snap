---
source: src/ui/help.rs
expression: terminal.backend()
snapshot_kind: text
---
"Press esc to exit,                                                              "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
"                                                                                "
